ArduCopter/mode_acro.cpp

```diff
@@ -96,46 +96,30 @@ float ModeAcro::throttle_hover() const
 
 // get_pilot_desired_angle_rates - transform pilot's roll pitch and yaw input into a desired lean angle rates
 // returns desired angle rates in centi-degrees-per-second
-void ModeAcro::get_pilot_desired_angle_rates(int16_t roll_in, int16_t pitch_in, int16_t yaw_in, float &roll_out, float &pitch_out, float &yaw_out)
+void ModeAcro::get_pilot_desired_angle_rates(float roll_in, float pitch_in, float yaw_in, float &roll_out, float &pitch_out, float &yaw_out)
```

Comment:
Let's put a comment here too specifying what the range of the roll_in, pitch_in and yaw_in arguments.. or make it clear by renaming the arguments.